core/modules/qdisp/Executive.cc

```diff
@@ -125,7 +129,7 @@ void Executive::add(JobDescription const& jobDesc) {
         return;
     }
     if (_empty.exchange(false)) {
-        LOGS(_log, LOG_LVL_DEBUG, "Flag _empty set to false by jobId=" << jobQuery->getIdStr());
+        LOGS(_log, LOG_LVL_DEBUG, "Flag _empty set to false by QId" << jobQuery->getIdStr());
```

Comment:
Just checking, this message will appear as:

    Flag _empty set to false by QIdQI=12345:

Is this as intended?